package.json

```diff
@@ -1,57 +1,30 @@
 {
-  "name": "@sphereon/openid4vci-client",
-  "version": "0.3.7-unstable.0",
-  "description": "OpenID for Verifiable Credential Issuance (OpenID4VCI) client",
-  "main": "dist/main/index.js",
-  "types": "dist/main/index.d.ts",
-  "author": "Sphereon",
-  "license": "Apache-2.0",
-  "private": false,
+  "name": "@sphereon/openid4vci",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
   "scripts": {
-    "build": "run-p build:*",
-    "build:main": "tsc -p tsconfig.build.json",
-    "clean": "rimraf dist coverage",
-    "watch": "tsc -w",
-    "fix": "run-s fix:*",
-    "fix:prettier": "prettier \"{lib,tests}/**/*.ts\" --write",
-    "fix:lint": "eslint . --ext .ts --fix",
-    "test": "run-s build test:*",
-    "test:lint": "eslint . --ext .ts",
-    "test:prettier": "prettier \"{lib,tests}/**/*.ts\" --list-different",
-    "test:cov": "jest --ci --coverage && codecov",
-    "uninstall": "rimraf dist coverage yarn.lock package-lock.json node_modules"
+    "preinstall": "npx only-allow pnpm",
+    "clean": "pnpm -r clean",
+    "fix": "pnpm -r run fix",
+    "build": "pnpm -r build",
+    "test": "pnpm -r run test",
+
+    "client:build": "pnpm -F @sphereon/openid4vci-client build",
```

Comment:
This is not the correct way to do this. So remove all these :test and :build commands for modules

The workspace aldready builds these. So do not mention them seperately. If someone wants to build an individual package he/she can just go to that module and run the build there.